tests/test_restore_names.py

```diff
@@ -1,3 +1,43 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_function_return_none():
```

Comment:
Function doesn`t return anything at all, so it will return None anyway. Therefore, this test is not needed.